File: laravel-demo/app/Http/Controllers/DashboardsController.php
<?php

namespace App\Http\Controllers;

use App\User;
use App\Http\Controllers\Controller;

class DashboardsController extends Controller
{
    public function dashboard2()
    {
        return view('dashboards.dashboard-2', ['title' => 'Dashboard 2 - Dashboards']);
    }

    public function dashboard3()
    {
        return view('dashboards.dashboard-3', ['title' => 'Dashboard 3 - Dashboards']);
    }

    public function dashboard4()
    {
        return view('dashboards.dashboard-4', ['title' => 'Dashboard 4 - Dashboards']);
    }

    public function dashboard5()
    {
        return view('dashboards.dashboard-5', ['title' => 'Dashboard 5 - Dashboards']);
    }

    public function dashboard1()
    {
        return view('dashboards.dashboard-1', ['title' => 'Dashboard 1 - Dashboards']);
    }

}

File: aspnet-mvc-demo/AspnetMvcDemo/Controllers/DashboardsController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace AspnetMvcDemo.Controllers
{
    public class DashboardsController : Controller
    {

         public ActionResult Dashboard2()
        {
            return View("~/Views/Dashboards/Dashboard2.cshtml");
        }

         public ActionResult Dashboard3()
        {
            return View("~/Views/Dashboards/Dashboard3.cshtml");
        }

         public ActionResult Dashboard4()
        {
            return View("~/Views/Dashboards/Dashboard4.cshtml");
        }

         public ActionResult Dashboard5()
        {
            return View("~/Views/Dashboards/Dashboard5.cshtml");
        }

         public ActionResult Dashboard1()
        {
            return View("~/Views/Dashboards/Dashboard1.cshtml");
        }

    }
}
